src/components/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import { CartContext } from "@/context/CartContext";
```

Comment:
Please remove unused imports

Reply:
I've removed the Header.tsx file completely since it is unrelated to this feature